### PR TITLE
Nlog example

### PR DESCRIPTION
**Pull request by yyyogev**

### CODE REVIEW

File: docs/nlog.md [hunk 1]
```diff
@@ -52,16 +52,25 @@ If you configure your logging in an XML file, you need to register the assembly
...
+var config = new LoggingConfiguration();
+
+// Replace these parameters with your configuration
+var logzioTarget = new LogzioTarget {
+  Token = "<<SHIPPING-TOKEN>>",
+  LogzioType = "nlog",
+  ListenerUrl = "<<LISTENER-HOST>>:8071",
+  BufferSize = 100,
+  BufferTimeout = TimeSpan.Parse("00:00:05"),
+  RetriesMaxAttempts = "3",
```
Comment (snakefoot) [inline docs/nlog.md:65]: Pretty sure `RetriesMaxAttempts` is not a string-value. So it should not be `"3"`

File: docs/nlog.md [hunk 2]
```diff
@@ -52,16 +52,25 @@ If you configure your logging in an XML file, you need to register the assembly
...
+  RetriesInterval = TimeSpan.Parse("00:00:02"),
+  Debug = false,
+  // ProxyAddress = "http://your.proxy.com:port"
+};
+
+config.AddTarget("Logzio", logzioTarget);
+config.AddRule(LogLevel.Debug, LogLevel.Fatal, "Logzio", "*");
```
Comment (snakefoot) [inline docs/nlog.md:72]: Change these lines:

```c#
var logzioTarget = new LogzioTarget {
...
};
config.AddTarget("Logzio", logzioTarget);
config.AddRule(LogLevel.Debug, LogLevel.Fatal, "Logzio", "*");
```

Into these lines:

```c#
var logzioTarget = new LogzioTarget {
     Name = "Logzio"
     ....
}
config.AddRule(LogLevel.Debug, LogLevel.Fatal, logzioTarget);
```

File: docs/nlog.md (hunk 1 above)
Comment (yyyogev) [reply docs/nlog.md:65]: Thanks! missed that one

**Comment by yyyogev** [comment]
@snakefoot Thanks again for all your help!
I sent you an invite to be a collaborator

**Comment by snakefoot** [comment]
> I sent you an invite to be a collaborator

Thank you for the invitation, but already very busy reviewing and helping on the NLog-core-project. Prefer being hit-and-run hobby-programmer for other projects, like this one.

**Comment by yyyogev** [comment]
Just making it a little bit easier to offer you helpful suggestions :)

**Comment by snakefoot** [comment]
Think it is okay right now. BTW I know that I'm not the best compiler, but I think you are missing a comma after: Name = "Logzio"


Sent from my Sony Xperia


---- Yogev Mets wrote ----


Just making it a little bit easier to offer you helpful suggestions :)

**Pull request merged**